app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from __future__ import annotations
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.elves.druid import Druid
+from app.players.elves.elf_ranger import ElfRanger
+
+
+def calculate_team_total_rating(
+        team: list
+        [Druid | DwarfBlacksmith | DwarfWarrior | ElfRanger]
```

Comment:
You can use Player like inner type of list, because it is parent class Druid, DwarfBlacksmith, DwarfWarrior, ElfRanger and this function uses only `get_rating`, that is specified in Player
```suggestion
        [Player]
```